overview.html

```diff
@@ -523,6 +524,89 @@ <h2 id="J2EE">J2EE</h2>
     configuration of your JNDI provider to register this factory and reference this library.
 </P>
 
+<h2 id="Logging">Logging</h2>
+
+<P>
+    This library uses the {@link java.util.logging} package. Classes that log will register a
+    {@link java.util.logging.Logger} under their fully qualified class name.
+    By default java-cloudant does not apply any logging configuration so the system defaults will be
+    used.
+</P>
+
+<P>
+    There are two ways to configure {@link java.util.logging}. It can be configured via a properties
+    file or programmatically. These examples show configuration for writing to the console for FINE
+    level or more severe for all {@code com.cloudant*} classes. The JVM documentation contains more
```

Comment:
"FINE level and above" would be my preferred wording, and it matches the wording in the properties file.